src/main/scala/org/antipathy/scoozie/builder/WorkflowBuilder.scala

```diff
@@ -19,21 +20,26 @@ private[scoozie] object WorkflowBuilder {
   def build(config: Config): Workflow = {
 
     implicit val credentials: Option[Credentials] =
-      ConfigurationBuilder.buildCredentials(config.getConfig("workflow"))
+      ConfigurationBuilder.buildCredentials(config.getConfig(HoconConstants.workflow))
 
-    val name = config.getString("workflow.name")
-    val path = config.getString("workflow.path")
+    val name = config.getString(s"${HoconConstants.workflow}.${HoconConstants.name}")
+    val path = config.getString(s"${HoconConstants.workflow}.${HoconConstants.path}")
     val configuration: Configuration =
-      ConfigurationBuilder.buildConfiguration(config.getConfig("workflow"))
+      ConfigurationBuilder.buildConfiguration(config.getConfig(HoconConstants.workflow))
 
     val yarnConfig =
-      YarnConfig(config.getString("workflow.yarn-config.name-node"),
-                 config.getString("workflow.yarn-config.job-tracker"))
-
-    val transitions = TransitionBuilder.build(Seq(config.getConfigList("workflow.transitions").asScala: _*), yarnConfig)
-
-    val jobXml = if (config.hasPath("workflow.job-xml")) {
-      Some(config.getString("workflow.job-xml"))
+      YarnConfig(
+        config.getString(s"${HoconConstants.workflow}.${HoconConstants.yarnConfig}.${HoconConstants.nameNode}"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The string literal '${ .. }.${ .. }.${ .. }' appears 2 times in the file.](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)